rc/servers.kak

```diff
@@ -511,6 +511,21 @@ hook -group lsp-filetype-toml global BufSetOption filetype=toml %{
     }
 }
 
+hook -group lsp-filetype-typst global BufSetOption filetype=typst %{
+    set-option buffer lsp_servers %exp{
+        [tinymist]
+        root_globs = [".git", ".hg"]
+        args = ["lsp"]
+        settings_section = "tinymist"
+        [tinymist.settings.tinymist]
+        # See https://myriad-dreamin.github.io/tinymist/configurations.html
+        exportPdf = "onDocumentHasTitle"
+        formatterMode = "typstyle"
+        formatterPrintWidth = %opt{autowrap_column}
```

Comment:
we try to avoid hard dependencies on other plugins (even stdlib ones);
to avoid breakage if someone doesn't have `autowrap.kak` in their autoload, can you re-declare this option after this line:

`declare-option -docstring 'name of the client in which documentation is to be displayed' str docsclient`